rdkit/RDConfig.py

```diff
@@ -29,10 +40,13 @@
   if condaDir[0]=='':
     condaDir[0] = os.path.sep
   condaDir += ['share','RDKit']
-  _share = os.path.join(*condaDir)
-  RDDataDir=os.path.join(_share,'Data')
-  RDDocsDir=os.path.join(_share,'Docs')
-  RDProjDir=os.path.join(_share,'Projects')
+  RDBaseDir = os.path.join(*condaDir)
+  RDDataDir=os.path.join(RDBaseDir,'Data')
+  RDDocsDir=os.path.join(RDBaseDir,'Docs')
+  RDProjDir=os.path.join(RDBaseDir,'Projects')
+  RDBinDir=MissingCondaDir('bin')
+  RDDemoDir=MissingCondaDir('Demo')
+  RDCodeDir=MissingCondaDir('rdkit')
```

Comment:
I don't understand this one: there should be a $RDBASE/rdkit dir after a conda installation, that's where the python code you get with "from rdkit import Chem" lives.